roles/config-gnome/tasks/gnome.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: "Install additional packages for Gnome"
```

Comment:
This file should be split into 2+ files like: gnome_{{ ansible_distribution }}.yml
One for Fedora, another for CentOS,....
Then in .../roles/config-gnome/main.yml: change the include line as follows.
include_vars: "gnome_{{ ansible_distribution }}.yml"